src/main/java/frc/robot/scoreStates/ShootState.java

```diff
@@ -44,20 +60,49 @@ public Command driveToClosestOptimalPoint() {
                     return AutoBuilder.pathfindToPose(
```

Comment:
If we are right outside the shooting zone and opposite the Speaker, the pathfound path might not be long enough to rotate the robot. In both cases, we should run a turn command at the end. Find a way to do that without duplicating code.
This is actually relevant to every place we use pathfinding.

Reply:
10f6e80bbfd2fe62bc6a6e6ee932d886b75a55cb

Reply:
2b68ba1

Reply:
If we are inside the wing there is no pathfind, Also pathfinding does not break it just reaches the position regardless the rotation and then ends that's why there is a rotate command after the DTOP.

Reply:
> If we are inside the wing there is no pathfind

Thanks, missed that.